lib/std/fs/file.zig

```diff
@@ -279,12 +279,12 @@ pub const File = struct {
         }
         const times = [2]os.timespec{
             os.timespec{
-                .tv_sec = @divFloor(atime, std.time.ns_per_s),
-                .tv_nsec = @mod(atime, std.time.ns_per_s),
+                .tv_sec = math.cast(isize, @divFloor(atime, std.time.ns_per_s)) catch maxInt(isize),
+                .tv_nsec = math.cast(isize, @mod(atime, std.time.ns_per_s)) catch maxInt(isize),
```

Comment:
I don't understand this... is it possible that `isize` (max 2147483648) can't hold `ns_per_s`? (which should be 1e9, right?)

Reply:
Symmetry. The modulo cannot overflow, the division can (and will). Using `catch unreachable` may be better here.